Lib/pdb.py

```diff
@@ -416,6 +418,17 @@ def preloop(self):
 
     def interaction(self, frame, traceback):
```

Comment:
I think we should rename the argument `traceback` to something like `tb_or_exc` - it's internal so should not impact users.